styles/style.css

```diff
@@ -0,0 +1,161 @@
+/* styling the body */
+
+body {
+  overflow-x: hidden;
+  margin: -20px 0 0 0;
+}
+
+/* start main section start */
+
+main {
+  min-width: 800px;
+  width: 100%;
+}
+
+.main-image {
+  background: url(https://web.archive.org/web/20140228114747im_/http://images.apple.com/v/home/as/images/your_verse_hero.jpg) 50% 100% no-repeat;
+  color: white;
+  height: 540px;
+  text-align: center;
+  width: 100%;
+}
+
+nav {
+  display: flex;
+  justify-content: center;
+  width: 100%;
+}
+
+h1 {
+  font-size: 3em;
+  font-weight: 100;
+  padding: 200px 0 0 0;
+}
+
+.sub-heading {
+  font-family: 'Spartan', 'Lucida Sans Unicode', Geneva, Verdana, sans-serif;
+  font-size: 1.5rem;
+  font-weight: 0.1rem;
+}
+
+nav ul {
+  display: flex;
+  justify-content: center;
+  list-style-type: none;
+  margin-top: 85px;
+  position: absolute;
+  width: 100%;
+}
+
+button {
+  background-image: linear-gradient(rgb(78, 78, 78), gray);
+  border-color: rgb(155, 151, 151);
+  border-width: 0.1rem;
+  box-sizing: border-box;
+  color: white;
+  cursor: pointer;
+  height: 35px;
+  border-top: gray;
+  width: 110px;
+}
+
+button :hover {
+  background-image: linear-gradient(rgb(51, 51, 51), rgb(97, 95, 95));
+}
+
+.apple-icon {
+  border-top-left-radius: 4px;
+  border-bottom-left-radius: 4px;
+}
+
+.right {
+  align-items: center;
+  background-image: linear-gradient(rgb(78, 78, 78), gray);
+  border-radius: 0 5px 5px 0;
+  display: flex;
+  height: 35px;
+  width: 110px;
+}
+
+i {
+  font-size: 20px;
+}
+
+.right i {
+  background-image: linear-gradient(rgb(78, 78, 78), gray);
+  border-radius: 20px 0 0 20px;
+  color: white;
+  font-size: 14px;
+  margin-left: 10px;
+  padding: 3px;
+}
+
+.right input {
+  background-image: linear-gradient(rgb(78, 78, 78), gray);
+  border-style: none;
+  border-radius: 0 20px 20px 0;
+  height: 20px;
+  padding: 0;
+  width: 60px;
+}
+
+aside {
+  display: flex;
+  padding: 2px;
+}
```

Comment:
- [ ] you don't need the padding here. It creates a space above the aside section that shouldn't be there.